install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/control-focal

```diff
@@ -0,0 +1,15 @@
+Source: securedrop-app-code
+Section: web
+Priority: optional
+Maintainer: SecureDrop Team <securedrop@freedom.press>
+Homepage: https://securedrop.org
+Build-Depends: debhelper (>= 9), dh-python, python3-all, python3-setuptools, dh-systemd, dh-virtualenv
+Standards-Version: 3.9.8
+X-Python3-Version: >= 3.8
```

Comment:
Is the `X-Python3-Version` field actually required? A quick glance at the dh-virtualenv source indicates it's not really used anywhere.

Reply:
Removed.